src/OSPSuite.Core/Converter/v7_3/Converter710To730.cs

```diff
@@ -10,6 +10,8 @@ namespace OSPSuite.Core.Converter.v7_3
 {
    public class Converter710To730 : IObjectConverter,
       IVisitor<IParameterStartValuesBuildingBlock>,
+      IVisitor<IPassiveTransportBuildingBlock>,
```

Comment:
I had forgotten passive transports. Also there are some parameters under SimulationSettings! 
This should be enough to make MoBi conversion work